ssl/ssl_lib.c

```diff
@@ -6540,26 +6661,34 @@ int ssl_log_rsa_client_key_exchange(SSL_CONNECTION *sc,
         return 0;
     }
 
+#ifndef OPENSSL_NO_SSLKEYLOG
     /* We only want the first 8 bytes of the encrypted premaster as a tag. */
     return nss_keylog_int("RSA",
                           sc,
                           encrypted_premaster,
                           8,
                           premaster,
                           premaster_len);
+#else
+    return 0;
+#endif
```

Comment:
`ssl_log_rsa_client_key_exchange()` is called unconditionally in `tls_construct_cke_rsa()`, so this must return 1. However, I think you can just drop the OPENSSL_NO_SSLKEYLOG here.